src/Sulu/Bundle/AdminBundle/Resources/js/containers/Link/tests/overlays/ExternalLinkTypeOverlay.test.js

```diff
@@ -18,6 +18,18 @@ test('Render overlay with an undefined URL', () => {
             onTargetChange={jest.fn()}
             onTitleChange={jest.fn()}
             open={true}
+            options={
+                {
+                    displayProperties: [],
+                    resourceKey: '',
+                    targets: {
+                        '_blank': 'sulu_admin.link_blank',
+                        '_self': 'sulu_admin.link_self',
+                        '_parent': 'sulu_admin.link_parent',
+                        '_top': 'sulu_admin.link_top',
+                    },
```

Comment:
give the translation key into it feels like of strange for me. I don't think a developer should need to care about the translation at this point so maybe a list of targets would be better:


```suggestion
                    targets: ['_blank', '_self', '_parent', '_top'],
```